crucible-llvm/src/Lang/Crucible/LLVM/Translation.hs

```diff
@@ -331,6 +331,8 @@ instrResultType instr =
     L.Conv _ _ ty -> ty
     L.Call _ (L.PtrTo (L.FunTy ty _ _)) _ _ -> ty
     L.Call _ ty _ _ -> error $ unwords ["unexpected function type in call:", show ty]
+    L.Invoke (L.FunTy ty _ _) _ _ _ _ -> ty
+    L.Invoke ty _ _ _ _ -> error $ unwords ["unexpected function type in invoke:", show ty]
```

Comment:
Well `Invoke` is not `Call` and its type is packed differently iiuc.  Let's look at a simple example:

```
int h()
{
    throw 2;
}

int g()
{
    int x;
    try {
        x = h();
    } catch(int err) {
        x = err;
    }
    return x;
}

int test()
{
    int a = g();
    return a;
}
```

And the bitcode for those call and invokes is parsed into an AST of:

```
  (Invoke
    (FunTy (PrimType (Integer 32)) [] False)
    (ValSymbol (Symbol "_Z1hv"))
    []
    (Anon 4)
    (Anon 5))
```

vs

```
(Call
    False
   (PtrTo (FunTy (PrimType (Integer 32)) [] False))
   (ValSymbol (Symbol "_Z1gv"))
   [])
```

So invoke gives us a type of `FunTy (PrimType (Integer 32)) [] False` while call gives `PtrTo (FunTy (PrimType (Integer 32)) [] False)`.